web/src/main/java/org/mskcc/cbio/oncokb/api/pvt/PrivateSearchApi.java

```diff
@@ -48,5 +49,16 @@
         @ApiParam(value = "The search query, it could be hugoSymbol or entrezGeneId.", required = true) @RequestParam(value = "gene", required = false) String queryGene,
         @ApiParam(value = "The level of evidence.", defaultValue = "false") @RequestParam(value = "level", required = false) String queryLevel
     );
+
+    @ApiOperation(value = "", notes = "Find matches based on blur query.", response = Treatment.class, responseContainer = "LinkedHashSet", tags = "")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "OK")})
+    @RequestMapping(value = "/search/typeahead",
```

Comment:
typeahead is the directive used in public website. It's only used in the typahead search box and under private API path. Will rename it if needed later on.